src/StreamsAPI_Lambda/Repetitive.java

```diff
@@ -0,0 +1,106 @@
+package StreamsAPI_Lambda;
+
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class Repetitive {
+
+
+    public static void main(String[] args) {
+
+        Map<Integer, String> map = new HashMap<>();
```

Comment:
hm, I'm not sure what this one does. if it's the solution to
> Write a program to print all character repetitions in a String. Example: "google" should have output:
>     g: 2
>     o: 2
>     l: 1
>     e: 1

then a solution might be like this:

```
        char[] charArray = "google".toCharArray();
        Map<Character, Integer> map = new HashMap<>();
        for (char c : charArray) {
            if (map.get(c) == null) {
                map.put(c, 1);
            } else {
                int currentCounter = map.get(c);
                map.put(c, ++currentCounter);
            }
        }
        map.forEach((key, value) -> System.out.println(key + ": " + value));
```